skynet.yaml

```diff
@@ -0,0 +1,13 @@
+name: verify-github-actions
+description: Verify that the github actions run passed, this is needed to make pipelines pass without manual intervention
+contact: 'Frontend Frameworks Architecture / #support-frontend-architecture'
+image: drydock.workiva.net/workiva/skynet-images:3693290 # Uses the image from this branch: https://github.com/Workiva/skynet-images/pull/127
```

Comment:
If wanting to review the actual script it is located at the PR listed [here](https://github.com/Workiva/skynet-images/pull/127) ^